skbeam/core/accumulators/projector.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python
+
+from __future__ import division
+import numpy as np
+
+"""
+authors: Mikhail Dubrovin, TJ Lane, Christopher O'Grady
+"""
+
+
+class Projector(object):
+
+    def __init__(self, bin_values, nbins, weights=None, norm=True):
```

Comment:
I've thought about this and will argue somewhat to the contrary:  currently the user can specify the equivalent of vmin/vmax arguments (and indeed many more complex "masks") by setting elements of the "weights" argument to zero.  I suggest that adding vmin/vmax is a more restrictive way of doing the same thing and makes the interface less clean.  But happy to discuss, of course.  Perhaps on the call tomorrow?